packages/liveblocks-react-comments/src/components/Composer.tsx

```diff
@@ -203,6 +205,21 @@ function ComposerMentionSuggestions({
   ) : null;
 }
 
+function ComposerLink({ url, children }: ComposerRenderLinkProps) {
+  const href = toAbsoluteURL(url);
+
+  return (
+    <ComposerPrimitive.Link
+      href={href}
+      target="_blank"
+      rel="noopener noreferrer nofollow"
+      className="lb-composer-link"
+    >
+      {children}
+    </ComposerPrimitive.Link>
```

Comment:
The `children` here is apparently quite important. If I instead render `{url}` here, Slate.js throws this error when a link node is rendered. 

```Cannot resolve a Slate point from DOM point```

This might make the `renderLink` feature a little confusing for developers because if they do not render the `children` prop, they editor errors out. Might have to spend a little more time on this.

Reply:
Yeah `children` contains invisible DOM nodes required by Slate, but it can be solved (similarly to `Composer.Mention`) by having an intermediate "wrapper" component that passes the invisible children so that the user-provided component doesn't have to. https://github.com/liveblocks/liveblocks/blob/main/packages/liveblocks-react-comments/src/primitives/Composer/index.tsx#L160

Reply:
I went ahead and replicated what we do with `Composer.Mention`: https://github.com/liveblocks/liveblocks/pull/1131/commits/6edb9f43b26967410128645a69cb736b169ad09e#diff-a4264574a0ff5a9b06f3691aa0544b8e8825b8fd990403f6988f7d3563c67551R205-R207

Reply:
The change above didn't work because links are editable, so I reverted to passing `children`. But for consistency, we now pass a `children` prop instead of `text` in both `renderLink` props. That also means `<a {...props} />` is now possible given the prop naming.